context/app/main.py

```diff
@@ -77,6 +77,19 @@ def create_app(testing=False):
     app.register_error_handler(504, gateway_timeout)
     app.register_error_handler(500, any_other_error)
 
+    # Credit to https://stackoverflow.com/a/40365514
+    # for this way to redirect from slash-ending urls to plain.
+    app.url_map.strict_slashes = False
+    @app.before_request
+    def clear_trailing():
+        path = request.path
+        if path.endswith('/') and path != '/':
+            return redirect(path[:-1])
+        # This is a little sketchy, but it's another common typo:
+        # egrep 'route\(.*s['"'"'"]' context/app/routes_*
+        if path.endswith('s') and path not in ['/cells', '/services', '/collections', '/my-lists']:
```

Comment:
I'm a little concerned someone wouldn't know this was here or that I would forget about it. Perhaps the list of paths to not trim should come from the routes pages or similar?